types/ember__test-helpers/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for @ember/test-helpers 0.7
+// Type definitions for @ember/test-helpers 0.8
```

Comment:
This is actually referring to the `major.minor` version of `@ember/test-helpers` that these definitions are appropriate for (and there is no `@ember/test-helpers@0.8`).

See: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

This should probably be updated to the latest stable version of `@ember/test-helpers` (`1.5` as of right now).

Reply:
Ah, I see. I thought the types run their own versioning system as mostly all `@types/ember*` do not match their package versions.

Reply:
> This should probably be updated to the latest stable version of @ember/test-helpers (1.5 as of right now).

I disagree with this. It should be the _oldest version of the package that the types accurately describe_.

Reply:
Ah, you're right @mike-north! The upstream type changes were introduced in https://github.com/emberjs/ember-test-helpers/commit/40d263a2c06ee515ffe8d1c6a93045a5c1422a5d as part of https://github.com/emberjs/ember-test-helpers/pull/463 and first included in `1.1.0`. It's possible that this type change is functionally accurate for earlier versions, though.

Reply:
aha, so `1.1` would be the correct version.